packages/apps/docs/src/pages/docs/marmalade/concrete-policies/collection-policy.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Collection Policy
+description: Utilizing the Collection Pßolicy
+menu: Collection Policy
+label: Collection Policy
+order: 1
+layout: normal
```

Comment:
```suggestion
layout: full
```
in general other than main landing page all sub/inner pages layout is `full`. So, I think it's better to align with other pages unless if you've any specific reason to have `normal`. 

This applies to all files have layout type `normal`